nipoppy/trackers/run_tracker.py

```diff
@@ -192,23 +226,33 @@ def run(global_configs, dash_schema_file, pipelines, session_id="ALL", run_id="1
                         _df.loc[bids_id,"pipeline_endtime"] = UNAVAILABLE # get_end_time(subject_dir)
                 else:
                     logger.debug(f"{pipeline} output is expected based on manifest but not found for bids_id: {bids_id}, session: {session}")
-                    for name in status_check_dict.keys():                    
-                        _df.loc[bids_id,name] = INCOMPLETE
-                        _df.loc[bids_id,"pipeline_starttime"] = UNAVAILABLE
-                        _df.loc[bids_id,"pipeline_endtime"] = UNAVAILABLE
+                    for name in status_check_dict.keys():
+
+                        _df.loc[bids_id, name] = INCOMPLETE
+
+                        # check that all required datatypes are available
+                        for datatype in PIPELINE_REQUIRED_DATATYPES[pipeline]:
```

Comment:
This branching feels a bit odd - shouldn't the "availability" of participant be verified before checking for "incomplete" status?